src/main/java/dk/cngroup/lentils/service/FinalPlaceService.java

```diff
@@ -25,12 +26,14 @@ public FinalPlace save(final FinalPlace finalPlace) {
         return finalPlaceRepository.save(finalPlace);
     }
 
-    /**
-     * Get final state entity.
-     *
-     * assumption:
-     * there is only one line in the table
-     */
+    public boolean isWithinOneHourBeforeOpeningTime() {
```

Comment:
Za mě bych tuhle metodu s touhle logikou dal do té GameLogic služby...
Oddělit to tak od té entity FinalPlace. Moje úvava byla taková, že se sice ten čas ukládá ve FInalPlace, ale je to takový hodně implementační detail z pohledu hry.

Takže bych ji dal tam a pojmenoval to třeba něco jako allowPlayersToViewFInalPlace nebo co já vím. Něco takového.